### ISSUE TITLE
serpent: add support for 192 and 256 bit keys

### ISSUE THREAD

**Issue by newpavlov** [RustCrypto/block-ciphers#327]
Currently the `serpent` crate supports only 128 bit keys.

Requested in #315.

cc @Antidote1911

**Comment by randombit** [comment]
This seems to have been addressed at some point

**Comment by newpavlov** [comment]
Yes, you are right. The cipher can be initialized with alternative key sizes (from 16 to 32 bytes) using the `KeyInit::new_from_slice` method.
